jenkins/scripts/integration_test.sh

```diff
@@ -36,12 +36,7 @@ UPDATED_REPO="${UPDATED_REPO:-https://github.com/${REPO_ORG}/${REPO_NAME}.git}"
 UPDATED_BRANCH="${UPDATED_BRANCH:-master}"
 CAPI_VERSION="${CAPI_VERSION:-v1alpha3}"
 
-if [ "${IMAGE_OS}" == "Ubuntu" ]
-then
-  DEFAULT_HOSTS_MEMORY="${DEFAULT_HOSTS_MEMORY:-6144}"
-else
-  DEFAULT_HOSTS_MEMORY="${DEFAULT_HOSTS_MEMORY:-8192}"
-fi
+DEFAULT_HOSTS_MEMORY="${DEFAULT_HOSTS_MEMORY:-8192}"
```

Comment:
I guess we're not raw streaming :\

Reply:
No, that is not in place, but I have been looking into that, and we could just convert the image after downloading them. we'd need adjustments on the metal3-dev-env side. that's the next step.